cypress/unit/utils.cy.ts

```diff
@@ -0,0 +1,43 @@
+import { parseNumericValue } from '../../src/utils/utils';
+
+describe('parseNumericValue', () => {
+  it('returns null for empty string', () => {
+    expect(parseNumericValue('')).to.equal(null);
+  });
+
+  it('returns null for whitespace-only string', () => {
+    expect(parseNumericValue('   ')).to.equal(null);
+  });
+
+  it('parses valid positive integer', () => {
+    expect(parseNumericValue('42')).to.equal(42);
+  });
+
+  it('parses valid positive decimal', () => {
+    expect(parseNumericValue('3.14')).to.equal(3.14);
+  });
+
+  it('parses valid negative number', () => {
+    expect(parseNumericValue('-10')).to.equal(-10);
+  });
+
+  it('parses zero', () => {
+    expect(parseNumericValue('0')).to.equal(0);
+  });
+
+  it('trims whitespace before parsing', () => {
+    expect(parseNumericValue('  25  ')).to.equal(25);
+  });
+
+  it('returns null for invalid numeric string', () => {
+    expect(parseNumericValue('abc')).to.equal(null);
+  });
+
+  it('returns null for mixed alphanumeric string', () => {
+    expect(parseNumericValue('12abc')).to.equal(null);
+  });
+
+  it('returns null for special characters', () => {
+    expect(parseNumericValue('!@#')).to.equal(null);
+  });
+});
```

Comment:
I'm confused. We already have unit tests in vitest: https://github.com/neurobagel/annotation-tool/blob/311e0bbebca79063c7a8cb10450ef4f944301ef5/src/utils/util.test.ts

What is different about the tests you have here that makes it so you need to run them in cypress instead? I'd really rather not have 2 parallel unit test frameworks if we can at all avoid it